cvat/apps/engine/view_utils.py

```diff
@@ -111,3 +112,14 @@ def decorator(f):
         return f
 
     return decorator
+
+def parse_mask_to_poly_request_param(request) -> Optional[bool]:
+    """
+    Retrieves and parses the 'conv_mask_to_poly' parameter from the request.
+    """
+    from cvat.apps.dataset_manager.formats.transformations import MaskToPolygonTransformation
+    param_value = request.query_params.get(MaskToPolygonTransformation.PARAMETER_NAME, None)
```

Comment:
The parsing and default value logic was copied in 5 different places, and it doesn't belong to any of them. The updated version allows to control the input value (including none) in the format handler. If we supply a default value here, the logic will be duplicated once again.

Reply:
> The parsing and default value logic was copied in 5 different places, and it doesn't belong to any of them. The updated version allows to control the input value (including none) in the format handler.

Of course it is clear, but I still don't understand why we cannot use True as the default value in 1 place here because in all subsequent functions to which this parameter is passed (I mean import_task_annotations/import_job_annotations/import_dataset_as_project), it is not expected that it will be None at least because it has the default value `True` in [_import_annotations](https://github.com/opencv/cvat/blob/5cad9356682a74af90c0248f34d266f1457934f3/cvat/apps/engine/views.py#L2821),  [_import_project_dataset](https://github.com/opencv/cvat/blob/5cad9356682a74af90c0248f34d266f1457934f3/cvat/apps/engine/views.py#L3048).
I prefer to use `True` as default value here and in this case, you don't need to handle `None` in `MaskToPolygonTransformation.convert_dataset` or migrate to optional `conv_mask_to_poly` with `None` in all places. But mixing the 2 approaches looks odd to me.

Reply:
Both `_import_annotations` and `_import_project_dataset` also duplicate the logic they do not own, so they probably need a fix then.

Actually, I noticed that some formats did not use the parameter and could benefit from it (e.g. COCO, Datumaro, ...). But the default value would change the default behavior for these formats, and there was no way to actually get the original input value in the format handler. So I decided only to refactor what we have now.